src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -250,7 +250,7 @@ export function LocationDetails({
                       {species.treeCount}{' '}
                       {species.scientificName
                         ? species.scientificName
-                        : species.otherSpecies}
+                        : species.otherSpecies?species.otherSpecies:t('unknown')}
```

Comment:
Maybe you can explicitly use `maps:unknown` or `treemapper:unknown` here.

Reply:
this should also be ok as also other resources are not using the prefix `treemapper:`